src/main/java/com/sun/caresyncsystem/model/entity/Booking.java

```diff
@@ -0,0 +1,36 @@
+package com.sun.caresyncsystem.model.entity;
+
+import com.sun.caresyncsystem.model.enums.BookingStatus;
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+
+@Entity
+@Getter
+@Setter
+public class Booking {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @ManyToOne(fetch = FetchType.LAZY)
```

Comment:
Mặc định fetch type của quan hệ many to one là gì

Reply:
mặc định là EAGER ạ